### PR TITLE
🐛Personalized-resource-limits: return default value when no product is defined

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
Fixes the exception raised when there is no product defined for a user. Instead of 500 error, a default permission (in this case a False) is returned.

### CODE REVIEW

**Review by odeimaiz — APPROVED** [review]
merci

File: services/web/server/src/simcore_service_webserver/users/_db.py [hunk 1]
```diff
@@ -45,16 +48,20 @@ async def list_user_permissions(
...
     engine = get_database_engine(app)
-
-    async with engine.acquire() as conn:
-        user_group_extra_properties = (
-            await GroupExtraPropertiesRepo.get_aggregated_properties_for_user(
-                conn, user_id=user_id, product_name=product_name
-            )
-        )
-    return [
+    permissions = [
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/users/_db.py:51]: MINOR: I have a style suggestion. 

```python
override_service_specs_permission = Permission(name="override_services_specifications", allowed=False)

async with engine.acquire() as conn:
      with contextlib.suppress(GroupExtraPropertiesNotFound):
            user_group_extra_properties = (
                await GroupExtraPropertiesRepo.get_aggregated_properties_for_user(
                    conn, user_id=user_id, product_name=product_name
                )
            )
           override_service_specs_permission.allowed = user_group_extra_properties.override_services_specifications

return [override_service_specs_permission, ]
```

**Review by pcrespov — APPROVED** [review]
Cool! Minor style suggestion

File: services/web/server/src/simcore_service_webserver/users/_db.py (hunk 1 above)
Comment (sanderegg) [reply services/web/server/src/simcore_service_webserver/users/_db.py:51]: ok

**Pull request merged**